packages/api/src/services/logging/logger.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "loggers": ["console"],
```

Comment:
It'd have been nicer if we'd used the new plugin api to plugin the logging services, but we could consider it for later stage.